airbyte-integrations/connectors/source-github/source_github/schemas/commit_comments.json

```diff
@@ -45,6 +45,9 @@
     },
     "author_association": {
       "type": ["null", "string"]
+    },
+    "reactions": {
+      "$ref": "reactions.json"
     }
```

Comment:
Do we know if we want to pass along this information with the commit_comments (and other things that can also report reactions as a column), when we allow users to configure streams for these specific reactions? 

In other words, if I were a user who decided I didn't want to sync the `commit_comment_reactions` stream, I might not be stoked that the reactions are still being stored in the `commit_comments` stream. This is just a thought and not an opinion either way - thought it was an interesting thing to think about. (One could probably say the same thing about their API, but you can't as explicitly opt out of information in that case)